etc/dbus-serialbattery/bms/lltjbd_ble.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+import asyncio
+import atexit
+import functools
+import threading
+from typing import Union, Optional
+
+from bleak import BleakClient, BleakScanner, BLEDevice
+
+from lltjbd import LltJbdProtection, LltJbd
+from utils import logger
+from struct import unpack_from
+
+BLE_SERVICE_UUID = "0000ff00-0000-1000-8000-00805f9b34fb"
+BLE_CHARACTERISTICS_TX_UUID = "0000ff02-0000-1000-8000-00805f9b34fb"
+BLE_CHARACTERISTICS_RX_UUID = "0000ff01-0000-1000-8000-00805f9b34fb"
+MIN_RESPONSE_SIZE = 6
+MAX_RESPONSE_SIZE = 256
+
+class LltJbdBle(LltJbd):
```

Comment:
Please rename to `LltJbd_Ble` since other checks use the `_Ble` to identify it as Bluetooth.